tool/healthz/main.go

```diff
@@ -79,13 +80,21 @@ func livenessProbe(s3Config *s3Config) error {
 		return trace.Wrap(err)
 	}
 
+	// change object name to be based on time.Now and POD hostname
+	now := time.Now()
+	hostname, err := os.Hostname()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	objectName := fmt.Sprintf("%s-%s-%v", defaultPrefix, hostname, now.Unix())
```

Comment:
I understood why you want to use time.Now as postfix, but then maybe we will try to delete all objects with prefix `defaultPrefix-pod_name`? To do not keep garbage of stale objects, if check fails after creation of object and do not clean it.